lib/i18n_rails_helpers.rb

```diff
@@ -28,4 +30,12 @@ def self.setup
   # Bootstrap icon class
   mattr_accessor :boot_icon_class_template
   @@boot_icon_class_template = 'icon icon-%s '
+
+  ActionController::Base.class_eval do
+    include I18nControllerHelpers
+  end
+
+  ActiveRecord::Base.class_eval do
+    extend I18nModelHelpers
+  end
```

Comment:
We should move these to [`lib/i18n_rails_helpers/engine.rb`](https://github.com/huerlisi/i18n_rails_helpers/blob/master/lib/i18n_rails_helpers/engine.rb) where `ActionView::Base` is extended as well.